__tests__/app.test.js

```diff
@@ -122,4 +130,58 @@ describe('GET /api/articles', () => {
 				expect(articles).toBeSortedBy('created_at', { descending: true });
 			})
 	})
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+
+	test('GET:200 should return an array of comments for the given article_id', () => {
+		return request(app)
+			.get('/api/articles/1/comments')
+			.expect(200)
+			.then((response) => {
+				const comments = response.body.comments;
+				expect(comments).toEqual(expect.any(Array));
+
```

Comment:
Please make sure that you also check that the length of the array is as expected

Reply:
done